apps/test/unit/code-studio/components/progress/teacherPanel/SelectedStudentInfoTest.jsx

```diff
@@ -72,27 +74,63 @@ describe('SelectedStudentInfo', () => {
     const levelWithProgress = {
       ...LEVEL_WITH_PROGRESS,
       paired: true,
+      isDriver: true,
       status: LevelStatus.perfect,
-      navigator: 'Student 2'
+      navigators: ['Student 2', 'Student 3']
     };
     const wrapper = setUp({levelWithProgress});
 
     expect(wrapper.contains('Worked With:')).to.equal(true);
     expect(wrapper.contains('Partner: Student 2')).to.equal(true);
+    expect(wrapper.contains('Logged in:')).to.equal(false);
+    expect(wrapper.contains('Last Updated:')).to.equal(true);
+  });
+
+  it('displays time and unknown navigator if paired as driver on level', () => {
+    const levelWithProgress = {
+      ...LEVEL_WITH_PROGRESS,
+      paired: true,
+      isDriver: true,
+      status: LevelStatus.perfect,
+      navigators: [null, 'Student 3']
```

Comment:
Now that I'm seeing this, it would be nice if the backend passed back the placeholder name. But we'd have to make that placeholder like `n/a` translatable.. It seems strange to pass an array around where null actually means something. Have you considered other options for this?

Reply:
Yeah, this is something I went back and forth on a couple of times.  I thought that it would be better for the back-end to return an indication that the data was missing and let the front-end decide how to display that.  I couldn't think of a better way to express "don't have this value" in JSON.  One possibility would be to pass a separate array of boolean flags (or similarly, return an array of objects which include the flag and the name) but that seemed too cumbersome?  Any ideas?

(In general, I'm not super comfortable with how much meaning I'm assigning to `nil` in this change.  I think using an [Option/Maybe pattern](https://medium.com/bettersharing/option-pattern-in-ruby-7b0f7c5abdb6) would be more robust but it feels kind of clunky in Ruby and I didn't want to introduce it as part of this change.)

Reply:
You could pass `annonymousNavigatorCount`? It's more data to pass around, but it more clearly communicates intent. I think having an array of nulls could easily confuse someone

Reply:
I ended up creating a separate method for (total) navigator count.  I like this much better than what I had originally, thanks for pushing on this!  Note that the count isn't fully plumbed through to the front-end yet.  I'll do that in a future PR after I discuss the UX with Amanda.

Reply:
Great solution!